pom.xml

```diff
@@ -920,7 +932,12 @@
                 <plugin>
                     <groupId>org.apache.cxf</groupId>
                     <artifactId>cxf-xjc-plugin</artifactId>
-                    <version>3.0.5</version>
+                    <version>3.3.0</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>3.8.1</version>
                 </plugin>
```

Comment:
Is upgrading the compiler to plugin to `3.8.1` necessary for this?  (It was previously at `3.8.0` - inherited from parent).